packages/docs/content/components/header/medicare-header.mdx

```diff
@@ -134,4 +134,4 @@ The Consistent Header has two breakpoints and three versions. At the lower range
   - Whenever you set lang="es" on the #root div the header expects the lang-toggle-link to be pointed to the english version of the site. So you would have to provide a link to the english version of the site.
   - Nothing else is needed on the translation front beyond the lang and lang-toggle-link
 - Is there anything that teams need to do to ensure the proper authenticated\unauth header is displayed to the user?
-  - No, there should not be anything that needs to be done from an integration standpoint for the header know the auth\unauth status of the user and this display the proper version of the header.
\ No newline at end of file
+  - No, there should not be anything that needs to be done from an integration standpoint for the header know the auth\unauth status of the user and this display the proper version of the header.
```

Comment:
This sentence is wonky and I'm not sure what needs to be changed to make it sound better

Reply:
Oh, prettier added a line break at the end of the file. I'm not going to change the sentence, since it's not even a component that we own.